frontend/src/components/dashboard/utils/charts.jsx

```diff
@@ -127,10 +126,20 @@ export const FeedsTypesChart = React.memo(() => {
     () => ({
       url: FEEDS_STATISTICS_TYPES_URI,
       accessorFnAggregation: (d) => d,
-      componentsFn: () =>
-        Object.entries(FEED_TYPE_COLOR_MAP).map(([dKey, color]) => (
-          <Bar stackId="feedtype" key={dKey} dataKey={dKey} fill={color} />
-        )),
+      componentsFn: (respData) => {
+        const [data] = respData;
+        if (!respData || !respData?.length) return null;
+        return Object.entries(data)
+          .slice(1)
+          .map(([dKey], i) => (
+            <Bar
+              stackId="feedtype"
+              key={dKey}
+              dataKey={dKey}
+              fill={colors[i]}
+            />
+          ));
```

Comment:
This is correct, but quite difficult to read:
* `const [data] = respData` I don't get this: i test with some data in jan and some in feb, everything is plotted correctly, but data contains only the jan. How is this possible ? Please add a comment
* slice remove an element from the list in the position passed as arg, it's correct because we don't want to plot the date field, but i suggest to make it clearer. We can delete it from the object before we convert it.
* I suggest to add a comment in the map: we need to use only the types/keys ([key] does this work) and also have the feed type mapped to a color, for this reason we need a index var (i). I suggest to add a comment to this or to find an easier way.